lib/matplotlib/contour.py

```diff
@@ -1029,23 +1038,6 @@ def _get_lowers_and_uppers(self):
         uppers = self._levels[1:]
         return (lowers, uppers)
 
-    def _make_paths(self, segs, kinds):
-        """
-        Create and return Path objects for the specified segments and optional
```

Comment:
The information here could be useful for developers, so I would prefer at least some of it to be kept in a comment. Perhaps near the new `allpaths` code above, or in `_get_allsegs_and_allkinds`.

Reply:
This is the comment above defining `allpaths`, which I have now expanded a bit.